cmd/celestia/main.go

```diff
@@ -5,16 +5,19 @@ import (
 	"os"
 
 	"github.com/spf13/cobra"
-	"github.com/tendermint/spm/cosmoscmd"
 
 	"github.com/celestiaorg/celestia-app/app"
 	"github.com/celestiaorg/celestia-node/cmd"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
 func init() {
 	// This is necessary to ensure that the account addresses are correctly prefixed
 	// as in the celestia application.
-	cosmoscmd.SetPrefixes(app.AccountAddressPrefix)
+	cfg := sdk.GetConfig()
+	cfg.SetBech32PrefixForAccount(app.Bech32PrefixAccAddr, app.Bech32PrefixAccPub)
+	cfg.Seal()
```

Comment:
Isn't the app config already sealed at this point? cc @evan-forbes

Reply:
@Bidon15 did you try this against a recent core/app combo?

Reply:
we only seal them when we are init our cli. unless we are importing and calling that cli, then I don't think the config is sealed

Reply:
But what is the purpose of sealing then? I'd assume you can't modify the config afterwards.